api/routes/ldap.ts

```diff
@@ -53,8 +53,11 @@ export default async function router(schema: Schema, config: Config) {
             })
         }),
         res: Type.Object({
-            cert: Type.String(),
-            key: Type.String()
+            integrationId: Type.Union([Type.Integer(), Type.Null()]),
+            certificate: Type.Object({
```

Comment:
Can we rename this to `auth: ` to match instances of cert/key pairs being returned together? Server Patch API, Connection Post API, etc